lib/backburner/index.js

```diff
@@ -28,7 +28,7 @@ export default function Backburner(queueNames, options) {
 
   var _this = this;
   this._boundClearItems = function() {
-    clearItems();
+    clearItems.apply(_this, arguments);
```

Comment:
I believe, we need it, yes. If you look at the implementation it does...

```
function clearItems(item) {
  this._platform.clearTimeout(item[2]);
}
```

Without passing `arguments`, it throws an unhandled error because item isn't passed.